contributed/plots/README copy.md

```diff
@@ -0,0 +1,5 @@
+# Plot Descriptions
+
+## Correlation Functions from TXPipe with CosmoDC2_small
+
+These plots were made by running the cosmoDC2_v1.0_small catalog through TXPipe.  TXPipe is a pipeline to perform 3x2 pt. analysis or the joint analysis of galaxy clustering, cosmic shear and galaxy-galaxy lensing on DC2 data.  The plots included show output of the pipleline of the three correlation functions for the correlation of galaxy positions, galaxy shears and the cross correlation of galaxy position and shears.  A mock metacal catalog was generated from the input catalog using the stage input_cats.py in the pipeline that simulates noise and shear response. 
```

Comment:
I recommend linking to the TXPipe repository when you refer to TXpipe.

Also, minor typo, "pipleline" -> "pipeline"